stories/Utilities/FrostedImage/frosted-background.scss

```diff
@@ -8,7 +8,7 @@
   position: relative;
 
   &::before {
-    @extend %frosted-bg;
+    background: rgba(247, 247, 247, 0.8);
```

Comment:
@paboden , I'm aware it wasn't explicitly stated how to change opacity, in that case we expect you to come back for clarifications. For this specific ticket default opacity should be 90% instead of 80%, so final value for background property should be `rgba(247, 247, 247, 0.9)`

Reply:
rgba is updated